utils/pub-helpers.rkt

```diff
@@ -0,0 +1,135 @@
+#lang racket
+
+(provide
+ ;; (Union String ByteString) -> (Listof publication)
+ ;;
+ ;; Given the path portion of a URI for an author's XML file on DBLP some author as a string or
+ ;; byte-string (e.g. "/pers/xx/s/Shivers:Olin.xml"), returns a list of publication structs for all
+ ;; the publications on that DBLP page we are able to read (currently supports article, book, and
+ ;; inproceedings types).
+ scrape-dblp
+
+ ;; (Listof publication) output-port -> void
+ ;;
+ ;; Writes the given publications to the given port in a format that can easily be copied and pasted
+ ;; into publications.rkt
+ write-pubs)
+
+;; ---------------------------------------------------------------------------------------------------
+
+(require xml
+         net/http-client
+         "../publication-data.rkt")
+
+(define (write-pubs pubs port)
+  (for ([pub pubs])
+  (match-define (publication title authors venue year link) pub)
```

Comment:
Because I'm not writing them out for serialization: I'm writing *exactly* what I want to copy and paste back into publications.rkt. If there's a better way to make Racket print/write/display a struct without tacking on the "#s# at the beginning, I'll gladly use it.